README.md

```diff
@@ -0,0 +1,65 @@
+# Local development
+## Setup 
+1. [Install Docker](https://docs.docker.com/install/) 
+1. [Install Docker Compose](https://docs.docker.com/compose/install/)
+1. [Install 10updocker](https://10up.github.io/wp-local-docker-docs/installation/)
+1. Linux Only: Run the command below and restart
+    ```bash
+    sudo usermod -a -G docker $USER
+    ```
+1. Configure 10updocker
+    ```bash
+    10updocker configure
+    ```
+    When asked `What directory would you like WP Local Docker to create environments within? ` choose the directory above the ctoec repository. 
+1. Create new env
+    ```bash
+    10updocker create ctoec-test
+    ```
+1. Replace the new env with this repository
+    ```bash
+    rm -rf ctoec-test
+    mv ctoec ctoec-test
+    ```
+1. Load the sample data
+    ```bash
+    mysql -h 127.0.0.1 -u root -ppassword ctoec < backup
```

Comment:
MySQL needs to be installed.

Reply:
Using `ctoec.test` as a hostname, the database was created as `ctoec-test` so this should be:

```bash
mysql -h 127.0.0.1 -u root -ppassword ctoec-test < backup
```